pkg/filestore/writer_test.go

```diff
@@ -299,6 +315,91 @@ func Test_WriteKeypair(t *testing.T) {
 			},
 			expErr: false,
 		},
+		"keystore PKCS12 with defined file and password": {
+			testBundle: pkcs8Bundle,
+			meta: metadata.Metadata{
+				VolumeID:   "vol-id",
+				TargetPath: "/target-path",
+				VolumeContext: map[string]string{
+					"csi.cert-manager.io/issuer-name":              "ca-issuer",
```

Comment:
**issue (formatting,blocking)**: This file needs `go fmt`, as it improves readability for these map literals - suggest you check your IDE why this has not been applied